test/framework/options.py

```diff
@@ -2822,6 +2822,28 @@ def test_merge_pr(self):
         ])
         self.assertTrue(expected_stdout in stdout)
 
+        # test --merge-pr on a PR that has a lot of comments
+        # cfr. https://github.com/easybuilders/easybuild-framework/pull/2553
+        args = [
+            '--merge-pr',
+            '6677',
+            '-D',
+            '--github-user=%s' % GITHUB_TEST_ACCOUNT,
+        ]
+        stdout, stderr = self._run_mock_eb(args, do_build=True, raise_error=True, testing=False)
```

Comment:
Just a silly comment: Doesn't it make sense to set `do_build=False` and `testing=True`? I took a brief look at the `_run_mock_eb` code (and `eb_main`, and `main`), and I don't think it makes any difference, but still.....